src/lib/generator/writeCsv.js

```diff
@@ -1,24 +1,28 @@
 import fs from 'fs';
-import { join, dirname, resolve } from 'path';
+
 import _ from 'lodash';
 
-const __dirname = resolve(dirname(''));
+import { icdCodesJsonFilePath, icdCodesCsvFilePath } from '.';
 
-function writeCsvLibrary() {
-    /**@type {Array<App.CodeEntry>} */
-    const jsonLibrary = JSON.parse(
-        fs.readFileSync(join(__dirname, 'src/files/output/icd_codes.json'), 'utf8'),
-    );
+/**
+ * @param {App.CodeEntry[]} jsonLibrary
+ */
+export function mapCsv(jsonLibrary) {
     const head = 'code,description,hcc_v24,hcc_v28,is_specific,parent,children';
     const rows = _.map(jsonLibrary, (entry) => {
         const parent = entry.parent ? entry.parent.code : '';
         const children = entry.children ? _.map(entry.children, 'code') : '';
-        return `${entry.code},"${entry.description}",${entry.hcc_v24},${entry.hcc_v28},${entry.is_specific},${parent},${children},`;
+        return `${entry.code},"${entry.description}",${entry.hcc_v24 || ''},${entry.hcc_v28 || ''},${entry.is_specific},${parent},${children},`;
```

Comment:
Why are the HCC values being defaulted to empty strings?

Reply:
So that the csv is `description,,,true` and not `description,undefined,undefined,true`

Reply:
Oh, I didn't realize the `undefined.toString()` was `"undefined"`.